frontend/package.json

```diff
@@ -18,6 +18,7 @@
     "coverage": "jest --coverage .",
     "eslint": "eslint --ext .js,.jsx,.ts,.tsx,.json,.gql,.graphql --color",
     "lint": "NODE_OPTIONS=--max-old-space-size=4096 yarn eslint .",
+    "lint-stylelint": "./node_modules/.bin/stylelint --fix 'packages/ceph-storage-plugin/**/*.scss'",
```

Comment:
The `stylelint` plugins in VSCode and Vim automatically recognize the nearest `.stylelintrc.json` file (in our case, the one that's inside `ceph-storage-plugin`) and provides lint highlights based on that (see https://github.com/openshift/console/pull/8360#issuecomment-811701231).

Reply:
Changed this to `lint-stylelint-fix`.